src/systems/ec/implementations/react/components/gallery/src/GalleryOverlay.jsx

```diff
@@ -28,11 +28,19 @@ const GalleryOverlay = ({ overlay, item, className, ...props }) => {
 
       <section className="ecl-gallery__slider">
         <div className="ecl-gallery__slider-image-container">
-          <img
-            src={item.src}
+          {/* eslint-disable-next-line jsx-a11y/media-has-caption */}
+          <video
+            className="ecl-gallery__slider-video"
+            data-ecl-gallery-overlay-video
+            controls="controls"
+            poster={item.image}
             alt={item.alt}
+          />
+          <img
```

Comment:
why are both the video and the image displayed at the same time?

Reply:
they are not displayed at the same time, only one of them is displayed. I had to keep the image tag anyways, for backward compatibility

Reply:
![image](https://user-images.githubusercontent.com/3524389/60101804-b9b5de00-975c-11e9-9a76-172060af861a.png)

Reply:
yep, they are not both displayed. Video has a display: none

Reply:
wouldn't it be better to inject/remove the `<img>` or `<video>` elements dynamically instead of polluting the DOM with unnecessary elements? :)

Reply:
that would be tricky to keep retro compatibility here...